packages/client/hmi-client/docker/Dockerfile

```diff
@@ -1,14 +1,29 @@
+## Stage 1 : build with gradle builder image with native capabilities
+FROM node:18.12.1-alpine as builder
+WORKDIR /code
+COPY package.json /code
+COPY yarn.lock /code
+COPY packages/client/graph-scaffolder/package.json /code/packages/client/graph-scaffolder/
+COPY packages/client/hmi-client/package.json /code/packages/client/hmi-client/
+RUN yarn install
+
+COPY . /code
+RUN yarn workspace graph-scaffolder tsc --build
+RUN yarn workspace hmi-client build
```

Comment:
Isn't this step done by `makefile build-hmi-client`?

Reply:
it is/was, with this change Make file can be removed as this is now fully self contained
Makefile is left for convenience for local testing at the moment